server/views/pages/support/offenderDetail.njk

```diff
@@ -143,6 +143,48 @@
         </div>
     </div>
 
+    <details class="govuk-details" data-module="govuk-details">
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          View dates stored by Create and vary a licence
+        </span>
+      </summary>
+      <div class="govuk-details__text">
+<table class="govuk-table">
```

Comment:
The tab-spacing of the code in this table seems a bit off, as does the </div> at the end

Reply:
Just bumping this comment